.github/workflows/lint-code.yml

```diff
@@ -44,52 +44,45 @@ jobs:
       - name: Run Prettier --check
         run: prettier --check ${GITHUB_WORKSPACE}
 
-  PythonBlack:
+  Ruff:
     runs-on: ["self-hosted"]
     steps:
-      - uses: actions/checkout@v4
+      - name: Check out source-code repository
+        uses: actions/checkout@v4
 
-      - name: Check code lints with Black
-        uses: psf/black@stable
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+      - name: Install Ruff
+        run: |
+          python -m pip install --upgrade pip
+          pip install ruff
+      - name: Run Ruff
+        run: ruff check .
```

Comment:
```suggestion
        run: ruff check .
        
     - name: Run Ruff check
        run: ruff check create-lint-wf/nf-core-testpipeline
```

Reply:
This folder doesn't exist here, did you mean to run `ruff format`?

Reply:
sorry yes

Reply:
```suggestion
        run: ruff check .

     - name: Run Ruff format
        run: ruff format .
```